media/icons/icon.svg

```diff
@@ -1 +1 @@
1,-0.95006 -0.52607,-2.10407l-0.52538,-2.09819h1.04033h1.04033l0.96389,3.8555l0.96389,3.8555h-2.80356h-2.80356l0.3491,1.39292c0.192,0.7661 0.3494,1.39552 0.34977,1.39869c0.00028,0.00317 -1.17463,0.00847 -2.61111,0.01175l-2.61178,0.00597l1.83613,2.09819l1.83614,2.09819l-1.5929,-0.0014l-1.5929,-0.0014l-1.70441,-2.10229l-1.70442,-2.10229h-4.02007h-4.02006l-1.70442,2.10229l-1.70441,2.10229l-1.5929,0.0014l-1.5929,0.0014zM19.97994,21.89591v-1.75143h-1.76319h-1.76319l-0.00608,1.72792c-0.00334,0.95036 -0.00112,1.74122 0.00519,1.75746c0.00894,0.02341 0.37533,0.02829 1.76927,0.0235l1.75799,-0.00603zM31.54644,21.89591v-1.75143l-1.75731,-0.00603l-1.75731,-0.00603v1.76348v1.76349l1.75731,-0.00603l1.75731,-0.00603z"/><path id="path30 1 1" fill="#fff" stroke="#d90000" stroke-miterlimit="4" stroke-width="0" d="M15.22145,30.65895l1.83613,-2.09819l-2.61179,-0.00597c-1.43648,-0.00328 -2.61148,-0.00858 -2.61111,-0.01175c0.00028,-0.00317 0.15777,-0.63259 0.34977,-1.39869l0.3491,-1.39292h-5.60711l0.96389,-3.8555l0.96389,-3.8555h2.08066l-0.52538,2.09819c-0.28896,1.15401 -0.52569,2.10084 -0.52607,2.10407c-0.00028,0.00322 0.79254,0.00589 1.76205,0.00589h1.76274l0.0164,-0.06465c0.00903,-0.03556 0.24527,-0.97974 0.52499,-2.09819l0.50859,-2.03354l5.01127,-0.0235l-1.08047,-1.32827c-0.59426,-0.73055 -1.10165,-1.35736 -1.12754,-1.39292l-0.04707,-0.06465h2.07147l1.22439,1.3988l1.22439,1.3988h4.53068l1.22439,-1.3988l1.22439,-1.3988h2.07147l-0.04707,0.06465c-0.02588,0.03556 -0.53328,0.66237 -1.12754,1.39292l-1.08047,1.32827l5.01127,0.0235l0.50859,2.03354c0.27972,1.11845 0.51597,2.06264 0.52499,2.09819l0.0164,0.06465h1.76274c0.96951,0 1.76243,-0.00252 1.76205,-0.00589c-0.00028,-0.00322 -0.23711,-0.95006 -0.52607,-2.10407l-0.52538,-2.09819h2.08066l0.96389,3.8555l0.96389,3.8555h-5.60711l0.3491,1.39292c0.192,0.7661 0.3494,1.39552 0.34977,1.39869c0.00028,0.00317 -1.17463,0.00847 -2.61111,0.01175l-2.61178,0.00597l1.83613,2.09819l1.83614,2.09819l-3.18579,-0.0028l-1.70441,-2.10229l-1.70442,-2.10229h-8.04013l-1.70442,2.10229l-1.70441,2.10229l-3.18579,0.0028zM19.97994,21.89591v-1.75143h-3.52637l-0.00608,1.72792c-0.00334,0.95036 -0.00112,1.74122 0.00519,1.75746c0.00894,0.02341 0.37533,0.02829 1.76927,0.0235l1.75799,-0.00603zM31.54644,20.14448l-3.51462,-0.01206v3.52697l3.51462,-0.01206z"/><path id="base 1 1 1" fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1" d="M2,24c0,-12.15026 9.84974,-22 22,-22c12.15026,0 22,9.84974 22,22c0,12.15026 -9.84974,22 -22,22c-12.15026,0 -22,-9.84974 -22,-22z"/><rect width="44" height="22" x="2" y="2" fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1"/><path fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1" d="M2,24v-22h44v22z"/><path fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1" d="M2,24v-22h44v22z"/><path fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1" d="M2,24v-22h44v22z"/><path fill="none" stroke="none" stroke-miterlimit="10" stroke-width="1" d="M2,24v-22h44v22z"/></g></svg>
```

Comment:
In addition, you should likely split these changes up into individual Pull Requests. Changing both the Bluetooth driver and optimizing the images is making a bunch of changes that can be difficult to test in two different change-sets. I'd recommend splitting the changes up into two different branches.